pkg/api/validation/validation.go

```diff
@@ -145,6 +152,23 @@ func ValidateEndpointsSpecificAnnotations(annotations map[string]string, fldPath
 	return allErrs
 }
 
+func validateOwnerReference(ownerReference api.OwnerReference, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+	gvk := unversioned.FromAPIVersionAndKind(ownerReference.APIVersion, ownerReference.Kind)
```

Comment:
who is responsible for making sure the owner reference is even a group/version/kind that exists in the system? if bogus values are given, what will the garbage collector do?